solidity-mode.el

```diff
@@ -493,6 +493,24 @@ Cursor must be at the function's name.  Does not currently work for constructors
   (interactive)
   (solidity--start-gasestimate (thing-at-point 'symbol 'no-properties)))
 
+;;; Support for imenu
```

Comment:
Hey @atimchenko92. Thank you very much. My colleague @rakanalh who uses Imenu tested it and it works.

I have one request only. Can you add a changelog entry [here](https://github.com/ethereum/emacs-solidity/blob/d6c48a1cb64d3c8a825dc0d06c839f2cacd4d289/changelog.MD#nightly)? Something like:
"Added support for Imenu".

Reply:
Added entry to changelog as well. My pleasure! :)